forward/forward_test.go

```diff
@@ -35,7 +35,8 @@ func TestCloneRequest(t *testing.T) {
 	const rawPath = "/%E4%B8%9C%E6%96%B9Project"
```

Comment:
@oxtoacart I guess that's because `Opaque` doesn't contain host. By setting it, the RequestURI of the http request becomes `scheme:opaque[?query][#fragment]`, effectively drops the host portion.